docs/source/guides/statistical_insights.ipynb

```diff
@@ -90,29 +89,29 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dt.mutual_information()"
+    "df.ww.mutual_information()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "#### Available Parameters\n",
-    "`dt.mutual_information` provides two parameters for tuning the mutual information calculation.\n",
+    "`df.ww.mutual_information` provides two parameters for tuning the mutual information calculation.\n",
     "\n",
     "- `num_bins` - In order to calculate mutual information on continuous data, Woodwork bins numeric data into categories. This parameter allows you to choose the number of bins with which to categorize data.\n",
     "    - Defaults to using 10 bins\n",
     "    - The more bins there are, the more variety a column will have. The number of bins used should accurately portray the spread of the data.\n",
-    "- `nrows` - If `nrows` is set at a value below the number of rows in the DataTable, that number of rows is randomly sampled from the underlying data\n",
+    "- `nrows` - If `nrows` is set at a value below the number of rows in the DataFrame, that number of rows is randomly sampled from the underlying data\n",
     "    - Defaults to using all the available rows.\n",
-    "    - Decreasing the number of rows can speed up the mutual information calculation on a DataTable with many rows, but you should be careful that the number being sampled is large enough to accurately portray the data."
+    "    - Decreasing the number of rows can speed up the mutual information calculation on a DataFrame with many rows, but you should be careful that the number being sampled is large enough to accurately portray the data."
```

Comment:
We'll need to remember to add the `include_index` parameter here when we add it into the Accessor in #694

Reply:
Was going to update the issue, but see you already have it covered!